parsons/ngpvan/bulk_import.py

```diff
@@ -46,6 +46,29 @@ def get_bulk_import_job(self, job_id):
         logger.info(f'Found bulk import job {job_id}.')
         return r
 
+    def get_bulk_import_job_results(self, job_id):
```

Comment:
It might be nice to provide some polling functionality in addition to the quick check here. Could you add another method to the class that polls this method every X seconds and returns when it gets a non-`None` result (or after a specified timeout)?

What I did in Rock the Vote was have three methods when downloading data.. one that kicks off the export (and returns the job ID), one that will check for results, and one that wraps both by kicking off the job and then polling the results until the data was available to download.

It's a non-trivial amount of work (and so might be outside the scope of this PR), but what if the import jobs returned an object that had methods for getting results (rather than the job ID).. so we could have a `ImportResult` that has `get_results` and `wait_on_results` (or some such). It adds another object, which isn't necessarily Parson-ic, but it might make for a more intuitive API than having to call back into the Connector with the `job_id`..?

Reply:
Will file this as a separate issue